CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Change Log
-All notable changes to this project will be documented in this file.  
+All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 5.1.0 - 2023-01-19
```

Comment:
FYI: there were three rereleases in between, which we did not document well. I don't think that would really matter much for description as you have already covered it here.

Reply:
Thanks @hshukla, I will mention in the release notes that the prior releases were not listed.